hack/cluster-configs/ocp_traffic_flow_manifests/sriov-pod.yaml.j2

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
the project was renamed from `ocp-traffic-flow-tests` to `kubernetes-traffic-flow-tests`. It seems better to not stick to the old name.

I realize, that there is already a pre-existing `ocp-tft-config.yaml`. Let's ignore the inconsistency of that name for now.

But the new directory, I think it should just be called `traffic_flow_tests_manifests/`, or `tft_manifests/` or `traffic_flow_tests/manifests/` or `tft/manifests/`. I like `tft/manifests/` the most.

Note that we also have `hack/traffic_flow_tests.sh`, so it's already the case that not all our files related to k8s-tft are named with something "ocp".